Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPostCampaignViewControllerTests.swift

```diff
@@ -83,122 +90,6 @@ final class NoShippingPostCampaignViewControllerTests: TestCase {
     }
   }
 
-  func testView_PledgeContext_FixPaymentMethod_ErroredCard() {
```

Comment:
Fix payment method is not a flow that ever happens in the late pledge context, so not something we should or need to test. I think this test was accidentally copy-pasted from the pledge vc.